web/src/components/vis/MetaboliteTable.vue

```diff
@@ -34,17 +37,29 @@ export default {
 </script>
 
 <template lang="pug">
-v-data-table.elevation-1(:headers="headers", :items="items", disable-initial-sort,
-    item-key="Metabolite")
+v-data-table.elevation-1.main(:headers="headers", :items="items", disable-initial-sort,
```

Comment:
Not really worth changing here, but I tend toward more descriptive class names.  `metabolite-table` for instance.